mlir/include/mlir/Dialect/IRDL/IR/IRDLOps.td

```diff
@@ -634,7 +634,7 @@ def IRDL_AllOfOp : IRDL_ConstraintOp<"all_of",
     ```
 
     The above program defines a type `complex` inside the dialect `cmath` that
-    can has one parameter that must be 32-bit long and a float (in other
+    can have one parameter that must be 32-bit long and a float (in other
```

Comment:
```suggestion
    has one parameter that must be 32-bit long and a float (in other
```
Actually, I think `has` would be better than `can have` here, because the parameter is required. Otherwise, readers might think that it is optional. What do you think?

Reply:
Sure, I chose "can have" because it is used in lines 600 and 637, so I'll change them as well (since it's not optional)